test/jest/turn-off-warnings.js

```diff
@@ -16,6 +16,8 @@ const errorBlacklist = [
   /Warning: Unknown event handler property `%s`./,
   /No metadata harvested from package files, so you will not get app icons./,
   /Warning: Failed prop type: Invalid prop `icon` supplied to `Icon`/,
+  /Warning: Failed %s type: %s%s/,
```

Comment:
We should disable all prop types warnings in tests - better to fix tests to pass all required props and have correct prop types
Created a ticket to fix tests https://issues.folio.org/browse/UISAUTCOMP-7